native-libs/deps/recipes/mymonerocorecpp/mymonerocorecpp.recipe

```diff
@@ -39,9 +39,12 @@ build() {
 
     make
     cd ..
-    cp -f src/serial_bridge_index.hpp contrib/monero-core-custom/cryptonote_config.h $install_dir/include/
+    cp -R contrib/monero-core-custom/* $install_dir/include/
+    cp -R contrib/monero-core-custom/epee/include/* $install_dir/include/
+    cp -R contrib/monero-core-custom/contrib/libsodium/include/* $install_dir/include/
+    cp -f src/serial_bridge_index.hpp $install_dir/include/
```

Comment:
We have new headers in `serial_bridge_index.hpp` - we need to pull those includes.